mediawiki.go

```diff
@@ -327,7 +327,7 @@ func (m *MWApi) Upload(dstFilename string, file io.Reader) error {
 	if err != nil {
 		return err
 	}
-	if !(response.Upload.Result == "Success" || response.Upload.Result == "Warning") {
+	if !(response.Upload.Result == "Success") || response.Upload.Result == "Warning" {
```

Comment:
Mediawiki returns a non-failure warning on things like duplicate file uploads. If you come across a warning that is indeed leading to your issue (and we are suppressing it) please let me know.